telco-hub/configuration/reference-crs/optional/backup-recovery/objectBucketClaim.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: backup
+  namespace: default
```

Comment:
Can this be created in the open-cluster-management-backup namespace along with the other resources?

Reply:
in fact , in real scenario this is created outside of the cluster we are backing up. you dont want to backup the cluster information within the same cluster. i think i put a note stating that the bucket need to be outside of the Hub.